### ISSUE TITLE
Suggestion: Show the source from which fetched version

### ISSUE THREAD

**Issue by IlanBeloglovsky** [dvershinin/lastversion#35]
The package is capable of fetching version from multiple sources,  but it wont show what was the version source when using 
`output_format='version'` which is also the default.
I think it would be really good if the returned object also contained the source of the version

**Comment by dvershinin** [comment]
By sources, did you mean the locations vs specific sources in GitHub?

For GitHub specifically, it is either "graphql" or "feed'.
This is available in `type` field of the object returned by dict/json output format.  (e.g. when you use `--format json` when running `lastversion` CLI app, of invoking it from your python code with `lastversion.latest("mautic/mautic", output_format='json')`

I've added it now so it's obvious what actual project location was used (it may be ambiguous if a one-word keyword used, which triggers GitHub search).
This is available as `from` field of the object returned by dict/json output format

For the `--verbose` output of the CLI app, the same will be included prior to the version being printed, for example:

```
# lastversion php:7.0 --verbose
....
....
lastversion.lastversion - INFO - Located the latest release at: https://github.com/php/php-src via graphql mechanism
7.0.33
```

Would that be fine?

**Comment by dvershinin** [comment]
Released in 1.3.0

**Comment by IlanBeloglovsky** [comment]
> By sources, did you mean the locations vs specific sources in GitHub?
> 
> For GitHub specifically, it is either "graphql" or "feed'.
> This is available in `type` field of the object returned by dict/json output format. (e.g. when you use `--format json` when running `lastversion` CLI app, of invoking it from your python code with `lastversion.latest("mautic/mautic", output_format='json')`
> 
> I've added it now so it's obvious what actual project location was used (it may be ambiguous if a one-word keyword used, which triggers GitHub search).
> This is available as `from` field of the object returned by dict/json output format
> 
> For the `--verbose` output of the CLI app, the same will be included prior to the version being printed, for example:
> 
> ```
> # lastversion php:7.0 --verbose
> ....
> ....
> lastversion.lastversion - INFO - Located the latest release at: https://github.com/php/php-src via graphql mechanism
> 7.0.33
> ```
> 
> Would that be fine?

I'll be more specific, the package allow us to fetch version for libraries stored in many places (github, gitlab, etc.)
What I would want is to know from where exactly we got the version, i.e.: 
```
            version = lastversion.latest('facebook/react', output_format='version', pre_ok=False)
            print(version['source'])  # github
```

**Comment by dvershinin** [comment]
@IlanBeloglovsky released lastversion 1.3.1 which allows you to get that info.
Use `output_format='dict'`.
The dictionary returned will have `version` and `source` keys among others. See bottom of the README for details.
